tests/fixtures/ecma-version/11/dynamic-import/basic.result.js

```diff
@@ -364,7 +364,7 @@ export default {
             ]
         },
         {
-            "type": "Keyword",
+            "type": "Identifier",
             "value": "import",
```

Comment:
This also might be a bug in Acorn. It's probably difficult for a javascript tokenizer to always correctly figure out whether some identifier names are used as keywords or identifiers in the given context, but `import` in particular is a reserved word so it must always be a keyword? Also, in earlier versions of Acorn `import` was indeed a keyword, so it might be good to report this issue to Acorn.

Reply:
agreed, will report it later.

Reply:
looks fixed in the latest version.